src/framework/theme/components/popover/popover.directive.ts

```diff
@@ -212,6 +218,7 @@ export class NbPopoverDirective implements OnInit, OnDestroy {
       .subscribe((containerRef: ComponentRef<NbPopoverComponent>) => {
         this.containerRef = containerRef;
         this.patchPopoverContent(this.content);
+        this.patchPopoverContext(this.context);
```

Comment:
do we need two functions here? can be just `patchPopover(this.content, this.context)`

Reply:
fixed